api/v1beta1/redis_types.go

```diff
@@ -26,21 +26,20 @@ import (
 
 // RedisSpec defines the desired state of Redis
 type RedisSpec struct {
-	KubernetesConfig  KubernetesConfig             `json:"kubernetesConfig"`
-	RedisExporter     *RedisExporter               `json:"redisExporter,omitempty"`
-	RedisConfig       map[string]string            `json:"redisConfig"`
-	Resources         *corev1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
This was never referenced or used, as its within the `KubernetesConfig`

Reply:
This is being used here

https://github.com/OT-CONTAINER-KIT/redis-operator/blob/716e52b07bdc44d808512b291d5e7da5b91d527d/k8sutils/redis-cluster.go#L38

Reply:
I think there's confusion in this `Resources` - as its `redis.Resources` which isn't being used anywhere, but the `redis.kubernetesConfig.Resources` and `redis.redisExporter.Resources` etc. (similar for cluster and standalone.